src/main/java/it/pagopa/ecommerce/commons/documents/v1/TransactionClosureErrorEvent.java

```diff
@@ -14,16 +14,18 @@
 @Generated
 @NoArgsConstructor
 @ToString(callSuper = true)
-public final class TransactionClosureErrorEvent extends TransactionEvent<Void> {
+public final class TransactionClosureErrorEvent extends BaseTransactionClosureEvent {
```

Comment:
`TransactionClosureErrorEvent` cannot inherit from `BaseTransactionClosureEvent` because `TransactionClosureData` contains a `outcome` field with Nodo's response on `closePayment`, while in case of error we have no such response

Reply:
Fixed in https://github.com/pagopa/pagopa-ecommerce-commons/pull/48/commits/128e42ccc5dc56aa6a04e6c2c90665e311520e67